.pre-commit-config.yaml

```diff
@@ -19,4 +19,4 @@ repos:
         types_or: [c, c++, cuda]
         files: '^timemachine/cpp/src/'
 
-exclude: '\.pdb$|\.sdf$|\.proto$|\.xml$|/vendored/|^ff/params/|^parallel/grpc/'
+exclude: '\.pdb$|\.sdf$|\.proto$|\.xml$|/vendored/|^timemachine.ff/params/|^parallel/grpc/'
```

Comment:
nit: should this be
```suggestion
exclude: '\.pdb$|\.sdf$|\.proto$|\.xml$|/vendored/|^timemachine/ff/params/|^parallel/grpc/'
```
(I think the `.` is matching any character)

Reply:
Doh -- thanks for catching!